src/Sylius/Behat/Context/Setup/CurrencyContext.php

```diff
@@ -93,11 +93,17 @@ public function theStoreHasCurrency($currencyCode)
 
     /**
      * @Given the store has currency :currencyCode, :secondCurrencyCode
+     * @Given the store has currency :currencyCode and :secondCurrencyCode
+     * @Given the store has currency :currencyCode, :secondCurrencyCode and :thirdCurrencyCode
      */
-    public function theStoreHasCurrencyAnd($currencyCode, $secondCurrencyCode)
+    public function theStoreHasCurrencyAnd($currencyCode, $secondCurrencyCode, $thirdCurrencyCode = null)
```

Comment:
We can use variadics `(...$currencies)` (see e.g. `storeShipsTo` method in `GeographicalContext`).